diff --git a/node_modules/wrangler/wrangler-dist/cli.js b/node_modules/wrangler/wrangler-dist/cli.js
index 3a0f474..8bcfb40 100644
--- a/node_modules/wrangler/wrangler-dist/cli.js
+++ b/node_modules/wrangler/wrangler-dist/cli.js
@@ -150175,7 +150175,7 @@ async function startApiDev(args) {
       upstreamProtocol,
       localProtocol: args.localProtocol ?? configParam.dev.local_protocol,
       localUpstream: args.localUpstream ?? host,
-      localPersistencePath,
+      localPersistencePath: null,
       liveReload: args.liveReload ?? false,
       accountId: configParam.account_id ?? getAccountFromCache()?.id,
       assetPaths,
